lib/screens/add_plate_weights.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:wger/providers/plate_weights.dart';
+
+class AddPlateWeights extends StatefulWidget {
+  const AddPlateWeights({super.key});
+
+  @override
+  State<AddPlateWeights> createState() => _AddPlateWeightsState();
+}
+
+class _AddPlateWeightsState extends State<AddPlateWeights> {
+
+  @override
+  Widget build(BuildContext context) {
+    return Consumer<PlateWeights> (
+      builder:(context,plateProvider,child)=> Scaffold (
+        appBar: AppBar (
+          title: const Text('Enter Custom Plate Weights'),
```

Comment:
Perhaps "Select available plates" would be better. Also, all user-facing strings should be translatable (this also applies for the buttons below, for those there should be already strings).

For this, just add the string to `app_en.arb` and then you can just use them with `AppLocalizations.of(context).<yourKey>;` (sometimes the IDE gets a bit confused or doesn't catch up immediately and you'll have to insert the import manually or it will mark the key as not existing, even if it does)